torch/csrc/api/include/torch/nn/modules/sequential.h

```diff
@@ -0,0 +1,172 @@
+#pragma once
+
+#include <torch/nn/module.h>
+#include <torch/nn/modules/any.h>
+
+#include <ATen/Error.h>
+
+#include <cstdint>
+#include <memory>
+#include <string>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+namespace torch { namespace nn {
+
+/// A `Sequential` module is a container for any number of other modules. Its
+/// `forward()` method chains outputs to inputs and returns the final output.
+/// The `Sequential` class reference semantics.
+class Sequential : public CloneableModule<Sequential> {
+ public:
+  using Iterator = std::vector<std::shared_ptr<AnyModule>>::iterator;
```

Comment:
Why not `::iterator` (small case) as in the STL?

Reply:
All our types are capitalized